src/granite/settings.cr

```diff
@@ -2,16 +2,11 @@ require "logger"
 
 module Granite
   class Settings
-    property database_url : String? = nil
-    property logger : Logger
+    class_getter adapters = [] of Granite::Adapter::Base
```

Comment:
Why should these be changed from properties of a class to class properties? This feels like it will set Granite back a few steps in composability.

Reply:
I'm not sure which is the preferred way of doing this.  I assumed ```@@settings ||= Settings.new` was kind of a hack and this was the better method?

If not i can change it back if you'd like.